@xen-orchestra/web-core/lib/components/ui/table-pagination/UiTablePagination.vue

```diff
@@ -0,0 +1,207 @@
+<!-- v2 -->
+<template>
+  <div class="ui-table-pagination">
+    <span class="typo p3-regular label">
+      {{ $t('core.select.n-object-of', { from: startIndex, to: endIndex, total: totalItems }) }}
+    </span>
+
+    <div class="buttons-content">
+      <UiButtonIcon
+        :disabled="isFirstPage"
+        accent="info"
+        class="icon"
+        size="small"
+        :icon="faAngleDoubleLeft"
+        @click="goToFirstPage()"
+      />
+      <UiButtonIcon
+        :disabled="isFirstPage"
+        accent="info"
+        class="icon"
+        size="small"
+        :icon="faAngleLeft"
+        @click="prev()"
+      />
+      <UiButtonIcon
+        :disabled="isLastPage"
+        accent="info"
+        class="icon"
+        size="small"
+        :icon="faAngleRight"
+        @click="next()"
+      />
+      <UiButtonIcon
+        :disabled="isLastPage"
+        accent="info"
+        class="icon"
+        size="small"
+        :icon="faAngleDoubleRight"
+        @click="goToLastPage()"
+      />
+    </div>
+
+    <span class="typo p3-regular label">{{ $t('core.show-by') }}</span>
+    <div class="dropdown-wrapper">
+      <select v-model="pageSize" class="dropdown typo c3-regular" @change="updatePagination">
```

Comment:
Using `goToFirstPage` will be more meaningful.

```suggestion
      <select v-model="pageSize" class="dropdown typo c3-regular" @change="goToFirstPage">
```